<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>中集-人检系统</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/imgs/TitleLogo.png</normaloff>:/imgs/TitleLogo.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" rowspan="2" colspan="8">
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_main">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/imgs/background.png);</string>
       </property>
       <attribute name="title">
        <string>首页</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_14"/>
      </widget>
      <widget class="QWidget" name="tab_data_show">
       <attribute name="title">
        <string>数据显示</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout" rowstretch="1">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab_mul_channels">
           <attribute name="title">
            <string>复合通道</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Channel01</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Channel02</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Channel03</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Channel04</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Channel05</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_23">
              <property name="text">
               <string>Channel06</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_24">
              <property name="text">
               <string>Channel07</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLabel" name="label_25">
              <property name="text">
               <string>Channel08</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_26">
              <property name="text">
               <string>Channel09</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="label_27">
              <property name="text">
               <string>Channel10</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="label_28">
              <property name="text">
               <string>Channel11</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QLabel" name="label_29">
              <property name="text">
               <string>Channel12</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_30">
              <property name="text">
               <string>Channel13</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="label_31">
              <property name="text">
               <string>Channel14</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLabel" name="label_32">
              <property name="text">
               <string>Channel15</string>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QLabel" name="label_33">
              <property name="text">
               <string>Channel16</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_single_channel">
           <attribute name="title">
            <string>单一通道</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="1">
             <widget class="QLabel" name="label_camera_sts">
              <property name="text">
               <string>相机</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse</set>
              </property>
             </widget>
            </item>
            <item row="0" column="7">
             <widget class="QLabel" name="label_output_sts">
              <property name="text">
               <string>输出</string>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QLabel" name="label_arclight_num">
              <property name="text">
               <string>弧光:0</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QLabel" name="label_body_num">
              <property name="text">
               <string>人体:0</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_io_sts">
              <property name="text">
               <string>io模块</string>
              </property>
             </widget>
            </item>
            <item row="0" column="6">
             <widget class="QLabel" name="label_input_sts">
              <property name="text">
               <string>输入</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="label_fps">
              <property name="text">
               <string>FPS:0</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QComboBox" name="comboBox_channel_pick">
              <item>
               <property name="text">
                <string>通道1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>通道2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>通道3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>通道4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>通道5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>通道6</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>通道7</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>通道8</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0" colspan="8">
             <widget class="QLabel" name="label_single_img_show">
              <property name="text">
               <string>人体检测</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_params_set">
       <attribute name="title">
        <string>参数设置</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_13">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_9">
          <property name="title">
           <string>常规参数</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_6">
             <property name="title">
              <string>相机参数</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="2" column="1">
               <widget class="QLineEdit" name="lineEdit_3"/>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>ip</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>端口</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>用户名</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>密码</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEdit_2"/>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="lineEdit_4">
                <property name="text">
                 <string/>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QGroupBox" name="groupBox_7">
             <property name="title">
              <string>PLC参数</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="0" column="0">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>ip</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_5"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>端口</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEdit_6"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>超时时间</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="lineEdit_7"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QGroupBox" name="groupBox_8">
             <property name="title">
              <string>文件设置</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_10">
              <item row="1" column="0">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>图像记录地址</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>AI模型地址</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEdit_9"/>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_8"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_10">
          <property name="title">
           <string>AI解析设置</string>
          </property>
          <widget class="QGroupBox" name="groupBox_11">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>30</y>
             <width>261</width>
             <height>151</height>
            </rect>
           </property>
           <property name="title">
            <string>ROI</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_12">
            <item row="0" column="0">
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>\</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>X</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>Y</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>ori点1</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>ori点2</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="groupBox_12">
           <property name="geometry">
            <rect>
             <x>320</x>
             <y>40</y>
             <width>241</width>
             <height>131</height>
            </rect>
           </property>
           <property name="title">
            <string>置信度设置</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_15">
            <item row="0" column="0">
             <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
            </item>
           </layout>
          </widget>
          <widget class="QPushButton" name="pushButton_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>730</x>
             <y>100</y>
             <width>122</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>停止</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>602</x>
             <y>100</y>
             <width>122</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>运行</string>
           </property>
          </widget>
          <widget class="QGroupBox" name="groupBox">
           <property name="geometry">
            <rect>
             <x>749</x>
             <y>230</y>
             <width>143</width>
             <height>44</height>
            </rect>
           </property>
           <property name="title">
            <string>信号输出</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>否</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="geometry">
            <rect>
             <x>450</x>
             <y>230</y>
             <width>143</width>
             <height>44</height>
            </rect>
           </property>
           <property name="title">
            <string>人体检测</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>否</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="geometry">
            <rect>
             <x>599</x>
             <y>230</y>
             <width>144</width>
             <height>44</height>
            </rect>
           </property>
           <property name="title">
            <string>弧光检测</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>否</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>932</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
